### ISSUE TITLE
premise_field( 'date' )

### ISSUE THREAD

**Issue by mvallejo3** [vallgroup/Premise-WP#1]
When using type date. does not output value into field.

**Comment by francoisjacquet** [comment]
@mvallejo3 the value passed as an argument to `premise_field()` function is displayed on my side
![image](https://cloud.githubusercontent.com/assets/4723501/9278394/200a96c2-4276-11e5-90d0-6b1f38dc84e4.png)

Please also note that date field type is supported only by HTML5 compatible browsers (not IE <= 8).

**Comment by francoisjacquet** [comment]
I have just noticed that you are currently working on different versions of the files I have.

**Comment by mvallejo3** [comment]
Yes I am. Just ignore it for now. I will bring you up to speed next week. 




Thanks for checking though ;)!
